src/modules/taskCompleted.js

```diff
@@ -0,0 +1,13 @@
+import { finalTodo, storeItem } from './addEditRemove.js';
+
+// Set checkbox to true and save to local storage
+export const checkedBox = (index) => {
+  finalTodo[index].completed = true;
+  storeItem();
+};
```

Comment:
- After an item is marked as completed, the changes:
- [ ] The task lists after checking them, then reloading the webpage, return back to being unchecked.
- [ ] When I try to mark the second item added as completed, it is checked on the UI, but not in the local Storage
- [ ] When I cleared local storage, then added two items, three items appear in the local storage instead

Please see my video below for more explanation:
https://www.loom.com/share/0fa8b53f37e54d14aa99e1717b4d2ed7?sid=94eb51fc-33a1-4047-b628-9fa0ff599fbc